packages/vuetify/src/components/VList/VList.tsx

```diff
@@ -228,7 +228,7 @@ export const VList = genericComponent<new <T>(
             dimensionStyles.value,
             props.style,
           ]}
-          tabindex={ (props.disabled || isFocused.value) ? -1 : 0 }
```

Comment:
This was so the isn't an "invisible" tab stop on shift+tab

```
<v-btn>Button</v-btn>
<v-list>
  <v-list-item link>List</v-list-item>
</v-list>
```

Reply:
understood, I changed the approach using a different helper getTabbableChild that uses a modified version of focusableChildren and focusChild so that the VList tabindex doesn't need to change